docs/book/component-guide/orchestrators/azureml.md

```diff
@@ -0,0 +1,166 @@
+and use it to run your pipeline steps in AzureML. Check out 
+[this page](../../how-to/customize-docker-builds/README.md) if you want to 
+learn more about how ZenML builds these images and how you can customize them.
+
+## AzureML UI
+
+Each AzureML workspace comes equipped with an Azure Machine Learning studio. 
+Here you can inspect, manage, and debug your pipelines and steps.
+
+![AzureML pipeline example](../../.gitbook/assets/azureml-pipelines.png)
+
+Double-clicking any of the steps on this view will open up the overview page 
+for that specific step. Here you can check the configuration of the component 
+and its execution logs.
+
+## Settings
+
+The ZenML AzureML orchestrator comes with a dedicated class called 
+`AzureMLOrchestratorSettings` for configuring its settings and it controls
+the compute resources used for pipeline execution in AzureML.
+
+Currently, it supports three different modes of operation.
+
+### 1. Serverless Compute (Default)
+- Set `mode` to `serverless`.
+- Other parameters are ignored.
+
+### 2. Compute Instance
+- Set `mode` to `compute-instance`.
+- Requires a `compute_name`.
+  - If a compute instance with the same name exists, it uses the existing 
+  compute instance and ignores other parameters.
+  - If a compute instance with the same name doesn't exist, it creates a 
+  new compute instance with the `compute_name`. For this process, you can 
+  specify `compute_size` and `idle_type_before_shutdown_minutes`.
+
+### 3. Compute Cluster
+- Set `mode` to `compute-cluster`.
+- Requires a `compute_name`.
+  - If a compute cluster with the same name exists, it uses existing cluster, 
+  ignores other parameters.
+  - If a compute cluster with the same name doesn't exist, it creates a new 
+  compute cluster. Additional parameters can be used for configuring this 
+  process.
+
+Here is an example how you can use the `AzureMLOrchestratorSettings` to define 
+a compute instance:
+
+```python
+from zenml.integrations.azure.flavors import AzureMLOrchestratorSettings
+
+azureml_settings = AzureMLOrchestratorSettings(
+    mode="compute-instance",
+    compute_name="MyComputeInstance",
+)
+
+
+@pipeline(
+   settings={
+       "orchestrator.azureml": azureml_settings
+   }
+)
+def pipeline():
+    # YOUR PIPELINE CODE
+    ...
+```
+
+### Run pipelines on a schedule
+
+The AzureML orchestrator supports running pipelines on a schedule using 
+its `[JobSchedules](https://learn.microsoft.com/en-us/azure/templates/microsoft.automation/2023-11-01/automationaccounts/jobschedules?pivots=deployment-language-bicep)`. Both cron expression and intervals are supported.
+
+```python
+from zenml.config.schedule import Schedule
+
+# Run a pipeline every 5th minute
+pipeline.run(schedule=Schedule(cron_expression="*/5 * * * *"))
+```
+
+Once you run the pipeline with a schedule, you can find the schedule and 
+the corresponding run under the `All Schedules` tab `Jobs` in the jobs page
+on AzureML.
```

Comment:
Maybe worth adding a note that if you schedule a pipeline using ZenML/Azure, they'll need to delete the schedule/pipeline from within Azure itself if they need (i.e. you can't do this via ZenML)

Reply:
Good idea. Added it.